vowpalwabbit/core/src/reductions/automl_impl.cc

```diff
@@ -121,6 +124,7 @@ void interaction_config_manager::gen_interactions(uint64_t live_slot)
         if (exclusions.find(idx) == exclusions.end()) { interactions.push_back({idx1, idx2}); }
       }
     }
+    if (ccb_on) { ccb::insert_ccb_interactions(interactions, empty); }
```

Comment:
Can this be added outside of `if (interaction_type == "quadratic")` to apply to any set of interactions?

Reply:
done